crates/ruff_linter/src/rules/ruff/snapshots/ruff_linter__rules__ruff__tests__range_suppressions.snap

```diff
@@ -150,6 +150,44 @@ note: This is an unsafe fix and may change runtime behavior
 
 
 --- Added ---
+RUF100 [*] Unused suppression (unused: `E501`)
+  --> suppressions.py:46:5
+   |
+44 |     # Neither of these are ignored and warnings are
+45 |     # logged to user
+46 |     # ruff: disable[E501]
+   |     ^^^^^^^^^^^^^^^^^^^^^
+47 |     I = 1
+48 |     # ruff: enable[E501]
+   |
+help: Remove unused suppression
+43 | def f():
+44 |     # Neither of these are ignored and warnings are
+45 |     # logged to user
+   -     # ruff: disable[E501]
+46 |     I = 1
+47 |     # ruff: enable[E501]
+48 | 
+
+
+RUF100 [*] Unused suppression (unused: `E501`)
+  --> suppressions.py:48:5
+   |
+46 |     # ruff: disable[E501]
+47 |     I = 1
+48 |     # ruff: enable[E501]
+   |     ^^^^^^^^^^^^^^^^^^^^
+   |
+help: Remove unused suppression
+45 |     # logged to user
+46 |     # ruff: disable[E501]
+47 |     I = 1
+   -     # ruff: enable[E501]
+48 | 
+49 | 
+50 | def f():
+
```

Comment:
Let's add some more tests demonstrating some edge cases:

* Duplicate rule codes (it's okay to add a todo)
* Disabling `RUF100` 
* Overlapping ranges (`ruff: disable[E501` followed by another `ruff: disable[E501]`)
* Multiple codes where only one code is unused
* Multiple codes where only some codes are unused
* Multiple codes were all codes are unused